coalib/bearlib/languages/definitions/C.py

```diff
@@ -0,0 +1,21 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class C:
+    extensions = '.c', '.h'
+    comment_delimiter = '//'
+    multiline_comment_delimiters = {'/*': '*/'}
+    string_delimiters = {'"': '"'}
```

Comment:
i think chars are with ``'``, dont you need those too? what are these string delimiters used for anyway?